dali/operators/reader/loader/video/video_test_base.cc

```diff
@@ -66,8 +67,20 @@ void VideoTestBase::LoadFrames(
 }
 
 void VideoTestBase::CompareFrames(const uint8_t *frame, const uint8_t *gt, int size, int eps) {
-    for (int j = 0; j < size; ++j) {
-      ASSERT_NEAR(frame[j], gt[j], eps);
-    }
+  for (int j = 0; j < size; ++j) {
+    ASSERT_NEAR(frame[j], gt[j], eps);
+  }
 }
+
+void VideoTestBase::CompareFramesAvg(
```

Comment:
nitpick: the name sounds as if you were comparing the average of each frame. Maybe `AvgError` would do

Reply:
Done